docs/spec/distribution/state.md

```diff
@@ -0,0 +1,108 @@
+## State
+
+### Global
+
+All globally tracked parameters for distribution are stored within
```

Comment:
Thanks, addressed. Just a note about github and PR comments - I couldn't actually see the line you were referring to because the comment was left on the line above it.